### ISSUE TITLE
Error while compiling Godot "'strcasecmp'/'strncasecmp' has not been declared; did you mean 'strcasestr'/'wcsncasecmp'?"

### ISSUE THREAD

**Issue by Vikram1323** [godotengine/godot#31953]
Solus 4.0 Fortitude
![Screenshot from 2019-09-04 15-45-52](https://user-images.githubusercontent.com/38137398/64246812-243c8480-cf2b-11e9-8ede-e5e2981741ab.png)

**Comment by clayjohn** [comment]
@RevoluPowered Is this related to your recent Assimp work?

**Comment by Vikram1323** [comment]
@clayjohn @RevoluPowered 
Few weeks ago I make install mingw-64 (cross compiler) on my machine.
In short libraries created by make install messed everything up.
After hours, I deleted them and now I can build Godot easily.

Thanks.

**Comment by Vikram1323** [comment]
![Screenshot from 2019-09-08 01-11-14](https://user-images.githubusercontent.com/38137398/64479494-9c2dd780-d1d5-11e9-9111-6f9de033f748.png)

**Comment by RevoluPowered** [comment]
@clayjohn I have not changed that part of the library directly but, I however did update the library to the latest master. After upgrading the library I had to do a full rebuild using scons due to global defines added to scons.

@Vikram1323 did you just delete assimp from modules or are you still unable to build with this included?

**Comment by Vikram1323** [comment]
@RevoluPowered 
Before, figuring out what is real problem* I build after deleting assimp but now I can build with assimp.

* Solus (GNU/Linux) has no package for mingw-64(cross-compiler). So I tried to build it. What  up happening in my opinion was libraries with same names was causing this problem. So I deleted them one by one.
![Screenshot from 2019-09-04 13-15-42](https://user-images.githubusercontent.com/38137398/64487166-84e1ff00-d254-11e9-9195-93a1fc209361.png)
![Screenshot from 2019-09-04 13-16-59](https://user-images.githubusercontent.com/38137398/64487167-890e1c80-d254-11e9-905b-e64917945083.png)
